gooddata-sdk/gooddata_sdk/table.py

```diff
@@ -193,12 +215,255 @@ def _as_table(response: ExecutionResponse) -> ExecutionTable:
     return ExecutionTable(response=response, first_page=first_page)
 
 
+@define
```

Comment:
If it is possible, use frozen instead of defined. If it is not possible, stick to define.